packages/xchain-radix/src/client.ts

```diff
@@ -0,0 +1,465 @@
+        },
+      ],
+      to: [
+        {
+          to: depositAccount,
+          amount: baseAmount(withdrawAmount),
+          asset: { symbol: withdrawResource, ticker: withdrawResource, synth: false, chain: 'radix' },
+        },
+      ],
+      date: new Date(confirmed_at),
+      type: TxType.Transfer,
+      hash: intent_hash,
+      asset: { symbol: withdrawResource, ticker: withdrawResource, synth: false, chain: 'radix' },
+    }
+    return transaction
+  }
+
+  /**
+   * Creates a transaction using the SimpleTransactionBuilder, signs it with the
+   * private key and returns the signed hex
+   * @param params - The transactions params
+   * @returns A signed transaction hex
+   */
+  async transfer(params: TxParams): Promise<string> {
+    const networkId = this.network === Network.Mainnet ? NetworkId.Mainnet : NetworkId.Stokenet
+    const transactionConstructionUrl = `${this.getGatewayUrl()}${TRANSACTION_CONSTRUCTION_PATH}`
+    const fromAccount = await this.getAddressAsync()
+    try {
+      const constructionMetadataResponse = await axios.post(transactionConstructionUrl)
+      const builder = await SimpleTransactionBuilder.new({
+        networkId: networkId,
+        validFromEpoch: constructionMetadataResponse.data.ledger_state.epoch,
+        fromAccount: fromAccount,
+        signerPublicKey: this.privateKey.publicKey(),
+      })
+
+      const transaction = await builder
+        .transferFungible({
+          toAccount: JSON.parse((await this.prepareTx(params)).rawUnsignedTx)['toAccount'],
+          resourceAddress: JSON.parse((await this.prepareTx(params)).rawUnsignedTx)['resourceAddress'],
+          amount: JSON.parse((await this.prepareTx(params)).rawUnsignedTx)['amount'],
+        })
+        .compileIntent()
+        .compileNotarizedAsync(async (hash) => this.privateKey.signToSignature(hash))
+      return transaction.toHex()
+    } catch (error) {
+      throw new Error('Failed to transfer')
+    }
+  }
+  /**
+   * Submits a transaction
+   * @param txHex - The transaction hex build with the transfer method
+   * @returns - The response from the gateway
+   */
+  async broadcastTx(txHex: string): Promise<string> {
+    const url = `${this.getGatewayUrl()}${TRANSACTION_SUBMIT_PATH}`
+    const requestBody = {
+      notarized_transaction_hex: txHex,
+    }
+
+    try {
+      const response = await axios.post(url, requestBody)
+      return JSON.stringify(response.data)
+    } catch (error) {
+      throw new Error('Failed to broadcast tx')
+    }
+  }
+
+  /**
+   * Prepares a transaction to be used by the transfer method
+   * It will include a non signed transaction
+   * @param params - The transaction params
+   * @returns a PreparedTx
+   */
+  async prepareTx(params: TxParams): Promise<PreparedTx> {
+    if (params.asset == undefined) {
+      throw new Error('asset can not be undefined')
+    }
+    const transaction = {
+      toAccount: params.recipient,
```

Comment:
On reflection, possibly this is not just an account. Do we know what they will use this for? e.g. might they include the MayaRouter here?

Reply:
so my understanding is that this is just an account. This client is a way to interact directly with the radix network, hence no Maya specific things here like the MayaRouter address. here's [binance](https://github.com/radixdlt/xchainjs-lib/blob/master/packages/xchain-binance/src/client.ts#L385). The rest of the clients seem to follow the same pattern

Reply:
Perhaps on other networks, Asgard vaults are just accounts? hmm